packages/compass-e2e-tests/helpers/commands/wait-for-connection-result.ts

```diff
@@ -18,10 +18,10 @@ export async function waitForConnectionResult(
     // indicator that we are connected to the server
     // TODO(COMPASS-8023): wait for the specific connection to appear in the
     // sidebar and be connected
-    selector = TEST_COMPASS_WEB
-      ? '[data-testid="workspace-tab-button"][title=Databases]'
-      : TEST_MULTIPLE_CONNECTIONS
+    selector = TEST_MULTIPLE_CONNECTIONS
```

Comment:
whyyy

Reply:
I'm totally rewriting this, so just don't want unnecessary conflicts :)

Reply:
I just thought this order makes more sense, I can revert since you're rewriting this anyway

Reply:
It is fine either way - will be an easy conflict to resolve given that I'd just ignore it.